tests/e2e/e2e_setup_test.go

```diff
@@ -228,6 +241,46 @@ func (s *IntegrationTestSuite) runValidators(chainConfig *chainConfig, dockerRep
 	s.valResources[chainConfig.meta.Id] = make([]*dockertest.Resource, len(chainConfig.validators)-len(chainConfig.skipRunValidatorIndexes))
 	pwd, err := os.Getwd()
 	s.Require().NoError(err)
+	if isFork == true {
+		for i, val := range chainConfig.validators {
+			// Skip some validators from running during set up.
+			// This is needed for testing functionality like
+			// state-sunc where we might want to start some validators during tests.
+			s.T().Logf("changing %s validator genesis with index %d...", val.validator.Name, i)
+			genesis := fmt.Sprintf("%s/config/genesis.json", val.validator.ConfigDir)
+			byteValue, err := ioutil.ReadFile(genesis)
+			if err != nil {
+				panic(err)
+			}
```

Comment:
Same with other panics
```suggestion
                        s.Require().NoError(err)
```